src/pages/Quiz/General.jsx

```diff
@@ -0,0 +1,5 @@
+export default function General(){
+    return(
+        <div>General Quiz</div>
+    )
```

Comment:
I guess exporting this, and similar files below this can be achieved better using an index.js export structure.
Just a suggestion.

Reply:
Yes I have done this into different file